sklearnex/tests/test_memory_usage.py

```diff
@@ -97,7 +97,10 @@ def remove_duplicated_estimators(estimators_list):
     return estimators_map.values()
 
 
-BANNED_ESTIMATORS = ("TSNE",)  # too slow for using in testing on common data size
+BANNED_ESTIMATORS = (
+    "IncrementalLinearRegression",  # TODO fix memory leak issue
```

Comment:
Mention of failed memory leak tests range is required here.

Reply:
what do you mean by range?

Reply:
Also, please send me the ticket for the follow-on for the memory leak investigation/fix

Reply:
@icfaust the ticket should have been created and assigned to you, please check your JIRA